team-props/scripts/ci-mock-google-services-setup.sh

```diff
@@ -4,15 +4,15 @@
 
 echo '{
   "project_info": {
-    "project_number": "1234567894652",
+    "project_number": "012345678912",
     "firebase_url": "https://squanchy-mock.firebaseio.com",
     "project_id": "squanchy-mock",
     "storage_bucket": "squanchy-mock.appspot.com"
   },
   "client": [
     {
       "client_info": {
-        "mobilesdk_app_id": "0",
+        "mobilesdk_app_id": "1:012345678912:android:1337deadbeef",
```

Comment:
That is the reason for this change; `deadbeef` is just a throwback thing in [hexspeak](https://en.wikipedia.org/wiki/Hexspeak):

![image](https://user-images.githubusercontent.com/153802/28216697-d9a1a378-68b2-11e7-8b46-c040f54678f9.png)

I originally considered `cafebabe` as an homage to the JVM but then PC and all so I was all like whatever, let's go non-vegan instead, just for you @fourlastor <3